test/logflare_web/controllers/api/account_controller_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule LogflareWeb.Api.AccountControllerTest do
+  use LogflareWeb.ConnCase
+
+  setup do
+    [token: token] =
+      Application.get_env(:logflare, LogflareWeb.Plugs.EnsureSuperUserAuthentication)
+
+    {:ok, %{token: token}}
+  end
+
+  describe "create/2" do
+    test "returns 201 and the user information and a token to access the API", %{
+      conn: conn,
+      token: token
+    } do
+      email = Faker.Internet.email()
+
+      response =
+        conn
+        |> put_req_header("authorization", "Bearer #{token}")
+        |> post("/api/accounts", %{email: email})
+        |> json_response(201)
+
+      assert response["user"]["email"] == email
+      assert response["token"]
```

Comment:
we want to generate access token for the newly created user and use that for authentication instead of using `user.api_key` field. using `api_key` is quite inflexible and we can't allow scoping of permissions in the future.

Reply:
that is what we're doing in `lib/logflare_web/controllers/api/account_controller.ex` and we return it in the body as the `token` field

Reply:
👌